### PR TITLE
Add file and ignore-git-commit flags

### PR DESCRIPTION
**Pull request by stroebs**
Add two flags to deploy command:
- `-f` / `--file`: Input a custom `kubetools.yml` file path to deploy from
- `--ignore-git-changes`: Ignore uncommitted changes to git when deploying

### CODE REVIEW

File: kubetools/config.py [hunk 1]
```diff
@@ -50,10 +51,14 @@ def load_kubetools_config(
         dev (bool): filter config items by dev mode
     '''
 
-    possible_filenames = (
-        'kubetools.yml',
-        'kubetools.yaml',
-    )
+    if custom_config_file:
+        possible_filenames = (custom_config_file)
```
Comment (Fizzadar) [inline kubetools/config.py]: Missing comma on the end here?

**Review by Fizzadar — APPROVED** [review]
One bug (I think), otherwise LGTM!

File: kubetools/config.py (hunk 1 above)
Comment (stroebs) [reply kubetools/config.py]: Good catch, fixed.

**Comment by stroebs** [comment]
@Fizzadar I added the flag to the `config` command too for parity.

**Review by Fizzadar — APPROVED** [review]

**Pull request merged**